src/components/DateRange.js

```diff
@@ -43,16 +43,37 @@ class DateRange extends Component {
     } else {
       endDate = value;
     }
+
     // reverse dates if startDate before endDate
+    let hasBeenReversed = false;
```

Comment:
instead of keeping a bool for `hasBeenReversed`, can you expose a variable like below:
```js
let isStartDateSelected = focusedRange[1] === 0;
// then
if (isBefore(endDate, startDate)) {
      isStartDateSelected = !isStartDateSelected;
      [startDate, endDate] = [endDate, startDate];	      [startDate, endDate] = [endDate, startDate];
    }	    
}
```